lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -54,6 +54,15 @@ export async function updateArtifacts({
         },
       ],
     };
+    const needKustomize = updatedDeps.some(
+      (dep) => dep.managerData?.needKustomize
+    );
+    if (needKustomize) {
+      execOptions.toolConstraints?.push({
```

Comment:
```suggestion
      execOptions.toolConstraints!.push({
```
you should store the constraints array as separate variable, so you don't need the assertion.